rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section23/rule_23_1.json

```diff
@@ -55,7 +52,8 @@
                                                 "heating_system": {
                                                     "id": "HP Heating Coil 1",
                                                     "type": "HEAT_PUMP",
-                                                    "heatpump_low_shutoff_temperature": -8.888888888888857
+                                                    "heatpump_auxilliary_heat_type": "ELECTRIC_RESISTANCE",
+                                                    "heatpump_low_shutoff_temperature": 7.222222222222285
```

Comment:
This data should be heatpump_auxilliary_heat_high_shutoff_temperature, and the value is 40F

Reply:
ahh my mistake on the key value. And sure, I can lower it to 40 F. I wasn't sure if the `energize auxiliary heat only when the outdoor air temperature is less than 40 F` was inclusive so bumped it up to 45 F bit to make sure it'd flag a failure.

Reply:
Actually-- I'm starting to think I got this mixed up (this is why I never do tests right on the line, I get myself all confused). It looks like 40 F should pass. Sorry, let me make this update.